Lib/test/test_logging.py

```diff
@@ -4310,6 +4311,20 @@ def test_issue_89047(self):
             s = f.format(r)
             self.assertNotIn('.1000', s)
 
+    def test_issue_102402_100msecs(self):
```

Comment:
Could the name be more descriptive (but not too long)?

Reply:
LMK what you think of the new name.